pkg/service/auth_context.go

```diff
@@ -5,53 +5,78 @@ import (
 	"strings"
 
 	"github.com/3scale/authorino/pkg/config"
+	"github.com/3scale/authorino/pkg/config/internal"
 
-	"golang.org/x/net/context"
 	auth "github.com/envoyproxy/go-control-plane/envoy/service/auth/v2"
+	"golang.org/x/net/context"
```

Comment:
I had disabled this auto-sort alphabetically thing but I guess not worth fighting against the tide 😞. I'll re-enable mine.